threads/views.py

```
@@ -72,23 +71,64 @@ def get_reactions_count(self, request, *args, **kwargs):
-    queryset = Reply.objects.all()
-    serializer_class = ReplySerializer
+    def update(self, request, *args, **kwargs):
+        check_permission(request.user, self.get_object())
+        return super().update(request, *args, **kwargs)
+
+    def destroy(self, request, *args, **kwargs):
+        check_permission(request.user, self.get_object())
+        return super().destroy(request, *args, **kwargs)
 
 
+    @action(detail=True, methods=["get"],url_path="reactions")
```

Flake8 (7.2.0)

[error] too many blank lines (2)

(E303)